src/App.js

```diff
@@ -5,8 +5,8 @@ import { connect } from 'react-redux';
 import { fetchAllUsers } from './store/actions/users';
 import { loginRequest, registerUser, logout } from './store/actions/auth';
 
-import NavBar from './components/NavBar';
-import Logout from './components/Logout';
+import NavBar from './components/layout/navbar/NavBar';
```

Comment:
Please try to keep the components as flat as possible and avoid nesting. As a followup to that it's better to have components with longer(but descriptive) names than to have multiple separate directories.